test_scripts/TheSameApp/Security/002_Send_RPC_or_encripted_RPC_in_protected_unprotected_mode_apps_with_same_appName_diff_appID_from_2_devices.lua

```diff
@@ -0,0 +1,90 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Please rename the file to `002_Send_RPC_or_encrypted_RPC_in_protected_unprotected_mode_apps_with_same_appName_diff_appID_from_2_devices.lua`

Reply:
@ShobhitAd Done (see fixup commit: e54cd14a9bca263cda45a11e7aea3c6fe8db812b)